lib/archive/util_test.go

```diff
@@ -528,6 +528,44 @@ func TestReadLevelOneDirpec(t *testing.T) {
 
 }
 
+func TestReadLevelOneNoTrailingSlash(t *testing.T) {
```

Comment:
I want to note here for ease of review that this is the same test as the above test except that the target has no `/` so we cover both cases here now.